learn/react-native/wavepulse.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "WavePulse"
+id: "wavepulse"
+sidebar_label: "WavePulse"
+---
+---
+
+WavePulse is a feature in WaveMaker, designed to provide real-time diagnostics for applications. It helps developers to ensure that their applications are running smoothly by offering insights into various metrics and logs.
+
+WavePulse is particularly useful for maintaining high-performance applications, ensuring stability, and providing end-users with a seamless experience. It enhances the process of debugging by allowing you to debug the APK or IPA file of an application without the need of using or sharing any confidential information like source code or excess information.
+
+To access WavePulse, [click here](#https://apps.wavemakeronline.com/wavepulse/client).
+
+### Key Features
+
+- **Easy Debug using IPA and APK**: Uses IPA or APK file to debug rather than using the React Native Zip file of an application.
+- **Quick and Simple Setup**: Simplifies the process of setup as it requires only one tool, WavePulse and few steps to get started with it.
+- **Debugging on Real Devices**: Debugging can be done on any device given WavePulse is enabled in the application to be debugged.
+- **Importing and Exporting Debugging Session**: Debugging session can be easily exported or imported using WavePulse without having to share any irrelevant or confidential information.
```

Comment:
> without having to share any irrelevant or confidential information

We are not filtering the data to remove unnecessary. So, this is statement should be removed.